Example/SmileID/Home/HomeViewModel.swift

```diff
@@ -85,17 +130,38 @@ class HomeViewModel: ObservableObject,
             jobName: "Biometric KYC",
             didSubmitJob: didSubmitBiometricJob
         )
+        dataStoreClient.saveJob(
+            data: JobData(
+                jobType: .biometricKyc,
+                timestamp: Date.getCurrentTimeAsHumanReadableTimestamp(),
```

Comment:
do you want to format date when you save it to datastore or format it when dispaying? no hard preference though

Reply:
Ah yes. I plan using Date type to store the timestamp, one limitation have encountered is ability to sort by most recent jobs. This will be implemented in a separate PR. Thanks for pointing that out.